frontend/src/pages/Home/index.js

```diff
@@ -0,0 +1,102 @@
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+
+import OutlinedContainer from "../../components/ui/OutlinedContainer";
+import DifficultyButtonGroup from "../../components/ui/DifficultyButtonGroup";
+
+function Home() {
```

Comment:
Just a nitpick and for consistency, can you help name this component `HomePage` so we know it's a page component, thanks!